docker/docker_test.go

```diff
@@ -0,0 +1,62 @@
+package docker_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/solo-io/go-utils/docker"
+	"io/ioutil"
+	"os/exec"
+)
+
+const (
+	validImage   = "soloio/gloo:0.7.0"
+	invalidImage = "soloio/gloo:0.1.0"
+)
+
+var _ = BeforeSuite(func() {
+	exec.Command("docker", "image", "prune", "-af").Run()
```

Comment:
can you only prune images you pulled?
i have some cached images that I want to stay cached.